Makefile-tests.am

```diff
@@ -287,6 +293,17 @@ tests/%-symlink-stamp: % Makefile
 	ln -sf "$${real_bin}" tests/$*; \
 	touch $@
 
+# See above comment on binding the tests to be either installed or not.
+if ENABLE_INSTALLED_TESTS
+installed_test_scripts = $(_installed_or_uninstalled_test_scripts)
+installed_test_programs = $(_installed_or_uninstalled_test_programs)
+check-local:
+	echo "NOTE: Installed tests are enabled; to run them, make install, then: gnome-desktop-testing-runner -p 0 libostree/"
```

Comment:
Sorry, I don't think this is right for a couple reasons.

1. The installed and in-tree tests are not replacements for each other. The former can run tests that the latter can't while the latter allows normal developer testing. I'd hate to have to build twice so I could run `make check` and `gnome-desktop-testing-runner` so I could run the tests not available in-tree.

2. If you want your CI system to not run tests twice, just don't run `make check`. Only run the installed tests.